### ISSUE TITLE
Configure number of items in dropdown

### ISSUE THREAD

**Issue by davidhund** [dudyn5ky1/select-pure#43]
Is it possible to configure more than max four (4) items shown in the dropdown?
In other words: can the dropdown be of a larger size?

**Comment by dudyn5ky1** [comment]
Not at the moment, but it's relatively easy to do.

I think css variable will solve the issue here. 

Will try to bring the change published today.

**Comment by dudyn5ky1** [comment]
Actually, do you mind giving some context here? 

Would the css variable work for you, or should this value be dynamic/controller by a JS?

**Comment by davidhund** [comment]
@dudyn5ky1 In my case it would be enough to add another CSS Custom Property `--dropdown-item-count`.

In https://github.com/dudyn5ky1/select-pure/blob/51eb08afc2f1fbc7f05c9638b86903e0cac3ff11/packages/select-pure/src/styles/select.styles.ts#L79 you currently have `4` and `3` hardcoded:

```CSS
max-height: calc(var(--select-height, 44px) * 4 + var(--border-width, 1px) * 3);
```

.. But this could be:

```CSS
/* How many items to show in dropdown: */
--dropdown-item-count: 4;

/* .. */
max-height: calc(var(--select-height, 44px) * var(--dropdown-item-count, 4) + var(--border-width, 1px) * calc(var(--dropdown-item-count, 4) - 1));
```
I believe this would allow custom 'dropdown height'...

**Comment by dudyn5ky1** [comment]
Added `--dropdown-items` property (default: `4`). Published in `2.1.1-alpha.1`.
